cogs/customreactions.py

```diff
@@ -0,0 +1,1111 @@
+                    title = "Modify a custom reaction. React with the option you want"
+                    footer = "{} is currently modifying a custom reaction. \n".format(
+                        user)
+                description = "Should the reaction be sent in the DMs of the user who called the reaction " \
+                              "instead of the channel?\n" \
+                              "0️⃣ No\n" \
+                              "1️⃣ Yes"
+                embed = discord.Embed(title=title, description=description)
+                embed.set_footer(text=footer, icon_url=user.avatar_url)
+                current_options.clear()
+                await message.clear_reactions()
+                current_options.extend(["0️⃣", "1️⃣", "⏹"])
+                await message.add_reaction("0️⃣")
+                await message.add_reaction("1️⃣")
+                await message.add_reaction("⏹")
+                await message.edit(embed=embed)
+                reaction = None
+                try:
+                    react_sent_by_user_modifying = False
+                    manual_timeout = time.time() + 60
+                    while not react_sent_by_user_modifying:
+                        reaction, reaction_user = await self.bot.wait_for(
+                            'reaction_add',
+                            check=reaction_check_any_user,
+                            timeout=60)
+                        if reaction_user == user:
+                            react_sent_by_user_modifying = True
+                        else:
+                            if time.time() > manual_timeout:
+                                raise asyncio.TimeoutError
+
+                except asyncio.TimeoutError:
+                    if proposals:
+                        title = 'The modification of the custom reaction proposal timed out. ' \
+                                'Returning to list of reaction proposals...'
+                    else:
+                        title = 'The modification of the custom reaction timed out. ' \
+                                'Returning to list of current reactions...'
+                    embed = discord.Embed(title=title)
+                    await message.edit(embed=embed)
+                    conn.close()
+                    time.sleep(5)
+                    current_options.clear()
+                    await message.clear_reactions()
+                    return
+
+                current_options.clear()
+                await message.clear_reactions()
+                if reaction is None:
+                    return
+                elif reaction.emoji == "0️⃣":
+                    if dm == 0:
+                        if proposals:
+                            title = "Successfully kept current option! Returning to list of reaction proposals..."
+                        else:
+                            title = "Successfully kept current option! Returning to list of current reactions..."
```

Comment:
duplicated string literals – use variables and format as needed

Reply:
See https://github.com/idoneam/Canary/pull/273#issuecomment-656869764